bindings/java/src/test/java/org/openucx/jucx/UcpEndpointTest.java

```diff
@@ -279,6 +279,7 @@ public void run() {
         worker2.recvTaggedNonBlocking(dst1, 0, -1, new UcxCallback() {
             @Override
             public void onSuccess(UcpRequest request) {
+                assertNull(request.getNativeId());
```

Comment:
why?

Reply:
Need to make sure it was set to completed and set the correct recv size. There are reproducible examples pointed in the mailing list, where recv size didn't set when there's a progress thread:
https://github.com/petro-rudenko/ucx/blob/jucx/fix-request-completed-with-reproduce/bindings/java/src/main/java/org/openucx/jucx/examples/UcxReceiver.java
https://github.com/petro-rudenko/ucx/blob/jucx/fix-request-completed-with-reproduce/bindings/java/src/main/java/org/openucx/jucx/examples/UcxReceiver.java

Reply:
seems like this assertion means the underlying C object is destroyed (or not set yet)?
How is this checking that recv size is correct?